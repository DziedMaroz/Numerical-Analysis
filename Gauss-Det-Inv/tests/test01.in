5
2 3 0 0 0
4 5 0 0 0
1 7 1 1 1
-1 3 2 3 4
9 8 4 9 16
