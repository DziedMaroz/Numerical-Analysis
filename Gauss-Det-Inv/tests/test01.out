RUNNING ON TEST: tests/test01.in
det (A) =   -4.000
det(A)*A^-1:
   10.00   -6.00   -0.00    0.00   -0.00
   -8.00    4.00    0.00   -0.00    0.00
  144.00  -58.00  -24.00   14.00   -2.00
 -138.00   46.00   32.00  -24.00    4.00
   40.00  -10.00  -12.00   10.00   -2.00
